charts/posit-chronicle/ci/no-local-storage-values.yaml

```diff
@@ -0,0 +1,21 @@
+image:
+  tag: "latest"
+
+serviceaccount:
+  enabled: false
```

Comment:
Right?

```suggestion
  create: true
```

Reply:
nice catch, looks our ci runs aren't exactly all encompassing. i'll double check this locally as well,